osm2gtfs/core/configuration.py

```diff
@@ -23,14 +25,69 @@ def __init__(self, args):
 
         """
         # Load config file from argument of standard location
-        self.config = self._load_config(args)
+        self.data = self._load_config(args)
 
         # Define name for output file
         self.output = self._define_output_file(args)
 
         # Validate and prepare start and end date
         self._prepare_dates()
 
+        # Initiate variable for schedule source information
+        self._schedule_source = None
+
+    def get_schedule_source(self, refresh=False):
+        """Loads the schedule source information.
+
+        Loads a schedule source file from either a path or a url specified
+        in the config file
+
+        :return schedule_source: The schedule read from a file.
+
+        """
+
+        if 'schedule_source' not in self.data:
+            return None
+
+        else:
+            source_file = self.data['schedule_source']
+            cached_file = 'schedule-source-' + self.data['selector']
+
+            # Preferably return cached data about routes
```

Comment:
typo: shouldn't it say `cached data about schedule`?

Reply:
Yes, thanks for noting.